content/docs/use-cases/fast-data-caching-hub.md

```diff
@@ -12,8 +12,8 @@ DVC's built-in data <abbr>caching</abbr> lets you implement a simple and
 efficient storage layer globally -- for your entire team. This approach can help
 to
 
-- speed up data transfers from massive object stores currently on the cloud, or
-  move a growing dataset out of your machine without slowing things down.
+- speed up data transfers from massive object stores on the cloud, or share data
+  across multiple on-premise machines without slowing things down.
```

Comment:
`on-prem` - is not the right term, just machines is enough I think

Reply:
OK. I thought we wanted to emphasize a local/virtual network team environment but yeah on-prem is not the best term for that either. Removing for now

```suggestion
  across multiple machines without slowing things down.
```

Reply:
Or maybe "multiple team member environments"

Reply:
?

Reply:
multiple machines sounds good to me :)